### PR TITLE
Add django-viewflow recipe

### PR DESCRIPTION
**Pull request by cshaley**

### CODE REVIEW

**Comment by cshaley** [comment]
Merging [https://github.com/conda-forge/django-filter-feedstock/pull/1](https://github.com/conda-forge/django-filter-feedstock/pull/1) should resolve the current error

**Comment by cshaley** [comment]
@jakirkham I hear you are potentially a subject matter expert here! I have no idea how to fix the current errors (see appveyor and travis-ci -- circleci didn't rerun when i opened/closed).  Do you have any idea?
```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
- django-viewflow 1.0.1 py36_0
- python 3.6*
```

File: recipes/django-viewflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    - django-filter >=1.0
```
Comment (jakirkham) [inline recipes/django-viewflow/meta.yaml]: Admittedly the error is vague, but it means that one of these dependencies doesn't exist on Python 3.6. My guess would be `django-filter`. Would recommend raising an issue on that [feedstock]( https://github.com/conda-forge/django-filter-feedstock ) to add Python 3.6 packages.

**Comment by synapticarbors** [comment]
I just put in a pull request re-rendering the django-filter feedstock: 

https://github.com/conda-forge/django-filter-feedstock/pull/4

Hopefully this will fix the issues in this PR.

**Comment by cshaley** [comment]
I believe this build is now ready to merge/have its own feedstock and be built + pushed to conda forge!

File: recipes/django-viewflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    django web development toolchain. BPMN provides a standard notation, 
+    readily understandable by all business stakeholders. Viewflow bridges 
+    the gap between a picture and an executable, ready to use web application.
+  doc_url: http://docs.viewflow.io/
+  dev_url: https://github.com/viewflow/viewflow/
+
+extra:
+  recipe-maintainers:
+    - cshaley
+    - kmmbvnr
```
Comment (synapticarbors) [inline recipes/django-viewflow/meta.yaml]: @kmmbvnr -- Are you ok with being added as a co-maintainer of the django viewflow recipe on conda-forge? You were added by @cshaley

File: recipes/django-viewflow/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - django >=1.8
+    - django-filter >=1.0
+
+test:
+  commands:
+    - python -c "import django; from django.conf import settings; settings.configure(INSTALLED_APPS=['viewflow']); django.setup(); import viewflow; import viewflow.flow; import viewflow.flow.views; import viewflow.management; import viewflow.migrations; import viewflow.nodes; import viewflow.templatetags; import viewflow.frontend; import viewflow.frontend.templatetags"
+
+about:
+  home: http://viewflow.io/
+  license: GPL 3.0
```
Comment (synapticarbors) [inline recipes/django-viewflow/meta.yaml]: It looks like it should be `AGPL-3.0`.

Comment (cshaley) [inline recipes/django-viewflow/meta.yaml]: change made

File: recipes/django-viewflow/meta.yaml (hunk 2 above)
Comment (cshaley) [inline recipes/django-viewflow/meta.yaml]: I went ahead and removed him - there's no reason he needs to manage this too!

**Comment by synapticarbors** [comment]
@cshaley Thanks for all of your work on this. A feedstock for this package should be generated soon. Please have a look at the README for how to push future updates to the recipe once it appears in the repo and let us know if you have any questions.

**Pull request merged**